compiler/il/OMRILOps.hpp

```diff
@@ -1456,6 +1472,19 @@ class ILOpCode
 
       switch (op)
          {
+         case TR::fsqrt:
+         case TR::dsqrt:
+            return ILOpCode::createVectorOpCode(TR::vsqrt, vectorType);
+         case TR::imin:
+         case TR::lmin:
+         case TR::fmin:
+         case TR::dmin:
+            return ILOpCode::createVectorOpCode(TR::vmin, vectorType);
+         case TR::imax:
+         case TR::lmax:
+         case TR::fmax:
+         case TR::dmax:
+            return ILOpCode::createVectorOpCode(TR::vmax, vectorType);
```

Comment:
A critique on this entire switch statement:  wouldn't it be better for the `switch` to just determine the opcode and then have one single call to `createVectorOpCode` after the `switch`?  It would undoubtedly save some shared library size without the multiple dispatches.   `l2d` might be an exception to that, but it could just return the way it is implemented now.

Reply:
[Done](https://github.com/eclipse/omr/compare/b19956691dbadd3604f1615c1824f6a1aa547941..186b918683ada0f44d18b35348f475d12e86403c)